web/gds-admin-ui/src/helpers/api/apiCore.js

```diff
@@ -37,17 +40,24 @@ axios.interceptors.response.use(
     }
 );
 
-const AUTH_SESSION_KEY = 'hyper_user';
-
 /**
  * Sets the default authorization
  * @param {*} token
  */
 const setAuthorization = (token) => {
-    if (token) axios.defaults.headers.common['Authorization'] = 'JWT ' + token;
+    if (token) axios.defaults.headers.common['Authorization'] = 'Bearer ' + token;
     else delete axios.defaults.headers.common['Authorization'];
 };
 
+const setCookie = (cookie) => {
+    if (cookie) {
+        axios.defaults.headers['X-CSRF-Token'] = cookie
+    }
+}
+
+const AUTH_SESSION_KEY = '__SESSION_TOKEN__';
```

Comment:
Just so I'm clear, this is to store the access token in local storage? What is the difference between `sessionStorage` and `localStorage`?

Reply:
the `sessionStorage` is only available in the current tab you are using and will be automatically deleted after closing the tab.
the `localStorage` will keep the data until you delete it yourself.

Reply:
Perfect, thank you for the clarification.